web-console/Storage/OCS/OCS _Installation_Workflow.md

```diff
@@ -0,0 +1,119 @@
+* Converged Mode -- Local Cluster / Hyperconverged 
+* Independent Mode -- External Cluster (Ceph Cluster is deployed externally)
+
+### Converged Mode Design
+
+OperatorHub
+![OperatorHub operators view](img/OCS-operator-view.png)
+
+Installation
+![OperatorHub operators view](img/OCS-Install-step1.png)
+
+![OperatorHub operators view](img/OCS-Install-step2.png)
+
+After the subscription process, the user will see the subscribed operator page. 
+
+![subscription operators page](img/OCS_Subscription_page.png) 
+
+In the installed operator view the user can see the OCS operator installed. 
+
+![OperatorHub operators view](img/Installed_operators_list.png)
+
+
+# The OCS installation process
+OCS Overview page
+![OCS-View](img/OCS-View.png)
+
+Clicking on "create new" (OCS Service) Will start the following installation flow: 
+
+Based on the underlying platform, the user can select between “Create new nodes” (IPI) and “Use existing nodes” (UPI).
+* Create new nodes -- System (OCS) creates new nodes along with underlying storage devices/volumes in the cloud, i.e. AWS, Azure, Google
+    * Note: not applicable for VMware currently and not applicable for baremetal
+* Use existing nodes -- uses existing OCP nodes and existing storage devices/volumes (in AWS and VMware, and for baremetal), which Admin has to select
+
+
+
+## For AWS or Cloud Deployments (IPI and UPI)
+* “Create New Nodes” (IPI) option:
+In AWS, three m4.large EC2 nodes will be created with the appropriate storage node label, which would be used by OCS.
+
+A new bucket will be created automatically in the AWS S3 connected cloud, in the same region as the OpenShift.
+
+![create new](img/Create_new_OCS_00.png)
+
+* “Use Existing Nodes” (UPI) option:
+A new bucket will be created automatically in AWS S3 connected cloud, in the same region as the OpenShift. 
+
+![use existing](img/Create_new_OCS_01.png)
+
+This info note should be added in cloud deployment:
+
+![Info note](img/info.png)
+The provider name should be modified according to the Cloud Credentials operator.
+
+
+
+### Select Nodes:
+ * Admin needs to select 3 nodes to label with “node-role.kuberentes.io/storage=" "node.ocs.openshift.io/cluster=" to be used for the OCS cluster.
+ * For each node selected, all available devices will be selected on the node by default.  Admin can elect to change the device selection, i.e. to use a subset of the devices if he/she needs to.
+
+![choose devices](img/Create_new_OCS_01.png)
+
+* The user can choose the devices he wants to use on each node by clicking on the Devices cell for a given node allows the Admin user to view and modify the device selection.  By default, all available devices on the node will be presented.
+
+**Note: The device modal will vary based on platform:**
+* Baremetal -- local devices presented
+* AWS -- EBS volumes presented. Device list should include the (EBS volume) name, EBS Type, Used Capacity, Capacity/Size, Availability Zone, Status
```

Comment:
For AWS, we cannot show Total Capacity/Size, just the Used Capacity. Please remove the Total column.